fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Utils/NXSpeeds.js

```diff
@@ -560,4 +586,49 @@ class NXSpeedsUtils {
     ) {
         return _convertKTASToMach(_convertKCasToKTAS(Vc, T, p), T);
     }
+
+    /** @private */
+    static interpolateTable(table, alt) {
+        if (alt <= table[0][0]) {
+            return vmca[0][1];
+        }
+        if (alt >= table[table.length - 1][0]) {
+            table[table.length - 1][1];
+        }
+        for (let i = 0; i < table.length - 1; i++) {
+            if (alt >= table[i][0] && alt <= table[i + 1][0]) {
+                const d = (alt - table[i + 1][0]) / (table[i + 1][0] - table[i][0]);
```

Comment:
Is it intentional that `d` will be negative here?

Reply:
😨